docs/software/distributed_projects.md

```diff
@@ -0,0 +1,196 @@
+# Distributed Projects
+
+## Overview
+
+- [**Folding@Home - Use processing power to assist with disease research**](#foldinghome)
+- [**Tor Relay - Add a node to the Tor network**](#tor-relay)
+- [**YaCy - Decentralized open source search engine**](#yacy)
+- [**IPFS Node - Contribute to a decentralized internet**](#ipfs-node)
+
+??? info "How do I run **DietPi-Software** and install **optimised software**?"
+    To install any of the **DietPi optimised software** listed below run from the command line:
+
+    ```sh
+    dietpi-software
+    ```
+
+    Choose **Software Optimised** and select one or more items. Finally click on `Install`. DietPi will do all the necessary steps to install and start these software items.
+
+    ![DietPi-Software menu screenshot](../assets/images/dietpi-software.jpg){: width="643" height="365" loading="lazy"}
+
+    To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools/) section.
+
+[Return to the **Optimised Software list**](../../software/)
+
+## Folding@Home
+
+[Folding@home](https://foldingathome.org/home/) is a project focused on disease research.  
+The problems which are solved require so many computer calculations that help from the community is needed to find the cures!
+
+*By default, our installation will add you to the DietPi team group, however, you can change this in the web interface at any time.*
+
+![Folding@Home web interface screenshot](../assets/images/dietpi-software-distributed-projects-foldingathome.png){: width="550" height="424" loading="lazy"}
+
+=== "Access the web interface"
+
+    The web interface is accessible via the port **7396** on the machine running the computation, e.g. this could be:  
```

Comment:
I propose to delete the section "on the machine running the computation, e.g. this could be"